report-status-azure.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
It doesn't look like this script is used. Is that right?

Reply:
oh, yes, correct!